inc/common.inc.sh

```diff
@@ -1292,7 +1292,7 @@ function displayQuotedEnum () {
     if [ -z "$trimmed_list" ]; then
         echo
     else
-        local quoted_list="'<b>${trimmed_list// /</b>', '<b>}</b>'"
+        local quoted_list="'<b>${trimmed_list// /</b>\', \'<b>}</b>'"
```

Comment:
Sorry, I don't understand the reason. And these escaped quotes failed unit tests: https://travis-ci.org/Twenga/twgit/builds/205628394#L166